mmv1/products/workstations/WorkstationConfig.yaml

```
@@ -24,6 +24,15 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     "Workstations": "https://cloud.google.com/workstations/docs/"
   api: "https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs/create"
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  method_name_separator: ":"
+  fetch_iam_policy_verb: :POST
+  parent_resource_attribute: workstationConfig
```

Similar case here, but also it should match the Terraform field name rather than the API field name.

```suggestion
  parent_resource_attribute: workstation_config_id
```

---

Thanks, good to know. Fixed it.